### ISSUE TITLE
Missing keyboard background

### ISSUE THREAD

**Issue by rbrucemtl** [loxK/android_device_htc_hero#23]
this problem existed also in early 2.1 builds

**Comment by vorcigernix** [comment]
I think this is a feature...it is even in emulator.

**Comment by rbrucemtl** [comment]
I don't think so... what exactly is the benefit of having a blocky looking keyboard on the screen?  It's not like that on the Nexus or any other 2.2 official ROMS or screenshots I've found.  It's been fixed in CM 5 so CM 6 should be the same fix.
